qutebrowser/mainwindow/messageview.py

```diff
@@ -100,6 +99,10 @@ def _set_clear_timer_interval(self):
         """Configure self._clear_timer according to the config."""
         interval = config.get('ui', 'message-timeout')
         if interval != 0:
+            multimpy_by = len(self._messages) + 1
+            if multimpy_by > 5:
+                multimpy_by = 5
+            interval *= multimpy_by
```

Comment:
You could simply do `interval *= min(5, len(self._messages) + 1)`

Why the `+ 1` though?